packages/calcite-components/src/components/block/block.tsx

```diff
@@ -251,7 +251,7 @@ export class Block
   /**
    * Emits when the component's header is clicked.
    *
-   * @deprecated use `openClose` events: `calciteBlock[Before]Open` and `calciteBlock[Before]Close` instead.
+   * @deprecated use `openClose` events: `calciteBlockBeforeOpen` and `calciteBlockBeforeClose` instead.
```

Comment:
nitpick: Capitalize Use.

Reply:
Would this also apply to four events - `calciteBlockBeforeOpen`, `calciteBlockOpen`, `calciteBlockBeforeClose`, and `calciteBlockClose`?

Reply:
Yeah I think so, will mention them.